packages/opencensus-core/src/exporters/console-exporter.ts

```diff
@@ -61,19 +61,19 @@ export class ConsoleExporter implements types.Exporter {
    * @param rootSpans
    */
   publish(rootSpans: modelTypes.RootSpan[]) {
+    const result: string[] = [];
     rootSpans.map((root) => {
       const ROOT_STR = `RootSpan: {traceId: ${root.traceId}, spanId: ${
           root.id}, name: ${root.name} }`;
       const SPANS_STR: string[] = root.spans.map(
           (span) => [`\t\t{spanId: ${span.id}, name: ${span.name}}`].join(
               '\n'));
-      const result: string[] = [];
 
       result.push(
           ROOT_STR + '\n\tChildSpans:\n' +
           `${SPANS_STR.join('\n')}`);
-      console.log(`${result}`);
     });
-    return Promise.resolve();
+    console.log(`${result}`);
+    return Promise.resolve(`${result}`);
```

Comment:
Why do we return the stringified result?

Reply:
The stringified result is what we end up sending to `console.log()`. I think it makes sense to return a `string` here too, but returning just `result` works fine. Do you prefer `Promise.resolve(result)`? It's ok for me to change.